notebooks/worldcereal_v1_demo_custom_cropland.ipynb

```diff
@@ -345,78 +198,40 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# <a id='toc7_'></a>[Deploy custom model](#toc0_)\n",
+    "### 6. Deploy your custom model\n",
     "\n",
-    "Once trained, we have to upload our model to the cloud so it can be used for inference. Note that these models are only kept in cloud storage for a limited amount of time.\n"
+    "Once trained, we have to upload our model to the cloud so it can be used by OpenEO for inference. Note that these models are only kept in cloud storage for a limited amount of time.\n",
+    "\n"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "\u001b[32m2024-10-12 17:02:54.665\u001b[0m | \u001b[1mINFO    \u001b[0m | \u001b[36mworldcereal.utils.upload\u001b[0m:\u001b[36mdeploy_model\u001b[0m:\u001b[36m205\u001b[0m - \u001b[1mDeploying model ...\u001b[0m\n"
-     ]
-    },
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Authenticated using refresh token.\n"
-     ]
-    },
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "\u001b[32m2024-10-12 17:02:56.096\u001b[0m | \u001b[1mINFO    \u001b[0m | \u001b[36mworldcereal.utils.upload\u001b[0m:\u001b[36mdeploy_model\u001b[0m:\u001b[36m211\u001b[0m - \u001b[1mDeployed to: s3://OpenEO-artifacts/fd307620ba8a0a07c44a2dc28541b181d5c03cb4/2024/10/12/demo_cropland_TZA_custommodel.onnx\u001b[0m\n"
-     ]
-    }
-   ],
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
    "source": [
     "from worldcereal.utils.upload import deploy_model\n",
     "from openeo_gfmap.backend import cdse_connection\n",
     "\n",
-    "model_url = deploy_model(cdse_connection(), custom_model, pattern=\"demo_cropland_TZA\")"
+    "modelname = <your_model_name>\n",
```

Comment:
already wrap this in a string to make sure the user knows it's supposed to be a string?

Reply:
used input method instead, asking user for a modelname (not containing spaces)
The modelname is then also used for naming the output directory